components/button/floating-buttons.js

```diff
@@ -0,0 +1,236 @@
+import 'lit-media-query/lit-media-query.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ButtonMixin } from './button-mixin.js';
+import { buttonStyles } from './button-styles.js';
+import { labelStyles } from '../typography/styles.js';
+
+class FloatingButtons extends ButtonMixin(LitElement) {
+	static get properties() {
+		return {
+			/**
+			 * Whether always enable floating buttons. Note: by default, buttons may not float depending on
+			 * space available since floating buttons may result in poor user experience when view-port is small
+			 * (ex. phones).
+			 */
+			alwaysFloat: {
+				reflect: true,
+				type: Boolean,
+				attribute: 'always-float'
+			},
+
+			/**
+			 * Minimum height of view-port in order for buttons to float.
+			 */
+			minHeight: {
+				reflect: true,
+				type: String,
+				attribute: 'min-height'
+			},
```

Comment:
Despite that the old component did this, I think we can stop reflecting `min-height` and `always-float` -- we don't reference these attributes in the CSS so seems unnecessary.